src/index.html

```diff
@@ -6,9 +6,64 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+
+    <header class="head">
+      <a href="#home" class="logo">
+        <img
+        class="icon"
+        src="images/logo.png"
+        alt="logo_img">
+      </a>
+      <nav class="menu">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a
+            class="nav_link is-active"
+            href="#Apple">Apple</a>
+          </li>
+          <li class="nav_item">
+            <a
+            class="nav_link"
+            href="#Samsung">Samsung</a>
+          </li>
+          <li class="nav_item">
+            <a
+            class="nav_link"
+            href="#Смарфоны и телефоны">Смартфоны и телефоны </a>
+          </li>
+          <li class="nav_item">
+            <a
+            class="nav_link"
+            href="#Ноутбуки и компьютеры"
+            data-ua="hover">Ноутбуки и компьютеры</a>
+          </li>
+          <li class="nav_item">
+            <a
+            class="nav_link"
+            href="#Гаджеты">Гаджеты</a>
+          </li>
+          <li class="nav_item">
+            <a
+            class="nav_link"
+            href="#Планшеты">Планшеты</a>
```

Comment:
```suggestion
            <a
            class="nav_link"
            href="#Планшеты"
          >
            Планшеты
          </a>
```
fix your links style 😉